### PR TITLE
feat: initial admin decoding serverless function

### PR DESCRIPTION
**Pull request by chrismaree**
This PR adds an initial implementation of an admin decoding serverless function to the vercell serverless deployment. it can be used like this:

Query `<domain>/api/decode-admin` with a `GET` request with a body containing an array of `identifiers` and you'll get back decoded information for the admin request. 

This works on local deployment as well when running `yarn dev` against `localhost:3000/api/decode-admin`

Example: query `localhost:3000/api/decode-admin` with a `GET` of body `{"identifiers":["Admin 181"]}`.

get back: 

```
[
	{
		"data": "0x35821038000000000000000000000000000000000000000000000000000000000000000a0000000000000000000000000000000000000000000000000000000000000040000000000000000000000000000000000000000000000000000000000000000100000000000000000000000000000000000000000000000000000000000000200000000000000000000000005be04e53b465c6fd89ecff3d36ddf666d198e31a000000000000000000000000000000000000000000000000000000000000004000000000000000000000000000000000000000000000000000000000000000447e7e591d000000000000000000000000420000000000000000000000000000000000000600000000000000000000000000000000000000000000000004db73254763000000000000000000000000000000000000000000000000000000000000",
		"to": "0x94520d90A4EBaA98e5A7B8D6809463f65198C104",
		"value": {
			"type": "BigNumber",
			"hex": "0x00"
		},
		"decodedData": "Transaction is a cross-chain governance action proposal containing 1 transactions:
		{
			    "name": "relayGovernance",
		    "params": {
			        "chainId": "10",
		        "calls": [
			            {
			                "to": "0x5be04e53b465c6fD89ECfF3d36dDf666D198e31a",
		                "data": "0x7e7e591d000000000000000000000000420000000000000000000000000000000000000600000000000000000000000000000000000000000000000004db732547630000"
		            }
		        ]
		    }
		}
		Nested transactions: 
		{
			    "name": "setFinalFee",
		    "params": {
			        "currency": "0x4200000000000000000000000000000000000006",
		        "newFinalFee": {
			            "rawValue": "350000000000000000"
		        }
		    },
		    "to": "0x5be04e53b465c6fD89ECfF3d36dDf666D198e31a"
		}"
	}
]
```

### CODE REVIEW

File: vercel.json
Comment (chrismaree) [inline vercel.json:1]: taken from https://github.com/across-protocol/frontend-v2/blob/master/vercel.json

File: package.json
Comment (ryanwolhuter) [inline package.json:16]: These can rather be installed as `devDependencies`

Try delete them from here, `yarn` and then install again but with `yarn add -D`

File: vercel.json
Comment (ryanwolhuter) [reply vercel.json:1]: @evaldofelipe is this all good?

**Review by ryanwolhuter — COMMENTED** [review]
LGTM — let's just see what Evaldo says about `vercel.json` but I'm sure it's all good

File: vercel.json
Comment (evaldofelipe) [reply vercel.json:1]: yes sir! good to go!

**Comment by ryanwolhuter** [comment]
@chrismaree can I close this one?

File: package.json
Comment (chrismaree) [reply package.json]: done.

**Comment by chrismaree** [comment]
> @chrismaree can I close this one?

was waiting on your approval :)

**Review by ryanwolhuter — APPROVED** [review]
🔥

**Pull request merged**
